### PR TITLE
Show announcement for live conference

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-14125
affects: Student
release note: Added a banner for live conferences to the dashboard

### CODE REVIEW

**Comment by vanwagonet** [comment]
This should not cause problems with the api missing, but hasn't been tested with the actual api in place.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/ajgrfro) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fajgrfro)
teacher: [Link](https://rebrand.ly/ozmjbsu) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fozmjbsu)
parent: [Link](https://rebrand.ly/s8abkcx) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fs8abkcx)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



#### Release Note: 
Added a banner for live conferences to the dashboard

#### Affected Apps: Student

[MBL-14125](https://instructure.atlassian.net/browse/MBL-14125)

<details><summary>:x: XCTest failed: CoreTests/ConferenceListViewControllerTests/testLayout</summary>


    Assertion Failure: ConferenceListViewControllerTests.swift:77: XCTAssertEqual failed: ("nil") is not equal to ("Optional("course_1")")
    Assertion Failure: ConferenceListViewControllerTests.swift:78: XCTAssertEqual failed: ("nil") is not equal to ("Optional("1")")
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: rn/Teacher/src/modules/dashboard/account-notification-reducer.js [hunk 1]
```diff
@@ -76,4 +80,32 @@ export const accountNotifications = handleActions({
...
+  [AccountNotificationActions.refreshLiveConferences.toString()]: handleAsync({
+    pending: (state) => ({
+      ...state,
+      liveConferencesPending: state.pending + 1,
+      liveConferencesError: '',
+    }),
+    resolved: (state, { result: { data } }) => ({
+      ...state,
+      liveConferencesPending: Math.max(0, state.pending - 1),
+      liveConferences: data,
```
Comment (armstrongnate) [inline rn/Teacher/src/modules/dashboard/account-notification-reducer.js]: This needs to be `data.conferences`. Tried this Adrian's changes locally and the conferences list is nested inside a top-level `conferences` key.

```
{"conferences":[]}
```

**Comment by armstrongnate** [comment]
Simulator Screen Shot - iPhone 8 - 2020-03-26 at 12 26 56

I was able to test it with the canvas changes locally and it worked great! Active conferences showed, dismissed conference hid, and ended conferences also went away. Nice job!

**Pull request merged**